src/BlockTypes/CollectionStockFilter.php

```diff
@@ -132,18 +134,21 @@ class="wc-block-components-checkbox__input"
 										</svg>
 										<span class="wc-block-components-checkbox__label">
 											<?php echo esc_html( $stock_statuses[ $stock_count['status'] ] ); ?>
-											<?php
+
+											<?php if ( $show_counts ) : ?>
+												<?php
 												// translators: %s: number of products.
 												$screen_reader_text = sprintf( _n( '%s product', '%s products', $stock_count['count'], 'woo-gutenberg-products-block' ), number_format_i18n( $stock_count['count'] ) );
-											?>
-											<span class="wc-filter-element-label-list-count">
-												<span aria-hidden="true">
-													<?php echo esc_html( $stock_count['count'] ); ?>
-												</span>
-												<span class="screen-reader-text">
-													<?php esc_html( $screen_reader_text ); ?>
+												?>
+												<span class="wc-filter-element-label-list-count">
+													<span aria-hidden="true">
+														<?php $show_counts ? print( esc_html( $stock_count['count'] ) ) : null; ?>
+													</span>
+													<span class="screen-reader-text">
+														<?php esc_html( $screen_reader_text ); ?>
+													</span>
```

Comment:
The checkbox list has some styling issue that adds padding to the count inside parenthesis.

Reply:
@dinhtungdu actually the css is an :after/:before that adds content of "( " and " )" I think? so there is a gap. I really don't know why it was done that way, but I thought maybe I would just leave it for now since its ported from existing styles.

Reply:
Resolved [here](https://github.com/woocommerce/woocommerce-blocks/pull/11827/commits/f78484b8093cf5db3b28ef8f26526a47455a41c5)